src/effects.js

```diff
@@ -646,12 +657,12 @@ jQuery.each( {
 } );
 
 jQuery.timers = [];
-jQuery.fx.tick = function() {
+jQuery.fx.tick = function( timestamp ) {
 	var timer,
 		i = 0,
 		timers = jQuery.timers;
 
-	fxNow = jQuery.now();
+	fxNow = timestamp || getTimestamp();
```

Comment:
I could optionally change `getTimestamp()` here to `jQuery.now()` as timestamp will always be defined if `tick` is called by jQuery & `rAF` & `perfomance.now()` are supported. But this seems safer, esp. that `tick` is exposed.